ch_3_AppDev_Basic/3_Quick_Start.ipynb

```diff
@@ -71,23 +80,57 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "save_directory='./open-llama-3b-v2-bigdl-llm-INT4'\n",
+    "\n",
+    "model.save_low_bit(save_directory)"
+   ]
```

Comment:
also show load usage. Refer to Notebook 4.1 save/load section.

Reply:
> also show load usage. Refer to Notebook 4.1 save/load section.

Have updated.